conf/template/yarp_config_system.h.in

```diff
@@ -1,8 +1,11 @@
 // -*- mode:C++; tab-width:4; c-basic-offset:4; indent-tabs-mode:nil -*-
 
 /*
- * Copyright: (C) 2009 RobotCub Consortium
- * Author: Paul Fitzpatrick
+ * Copyright: (C) 2006-2009 RobotCub Consortium
+ *            (C) 2014 iCub Facility, Istituto Italiano di Tecnologia
```

Comment:
Good catch... it is still January, I usually realize that the new year started in June/July... :D